src/services/flowsheet.service.ts

```diff
@@ -1,21 +1,21 @@
-import { sql, desc, eq, and, lte, gte } from 'drizzle-orm';
+import { and, desc, eq, gte, lte } from 'drizzle-orm';
+import { IFSEntry, UpdateRequestBody } from '../controllers/flowsheet.controller';
 import { db } from '../db/drizzle_client';
 import {
+  DJ,
+  FSEntry,
   NewFSEntry,
-  flowsheet,
-  shows,
   Show,
-  // NewShow,
-  rotation,
-  library,
+  ShowDJ,
   artists,
-  DJ,
   djs,
-  FSEntry,
+  flowsheet,
+  library,
+  // NewShow,
```

Comment:
Is this intentionally left here? If so, could you add a todo comment for what needs to happen in order to uncomment this?